internal/context/prompts.go

```diff
@@ -15,17 +16,34 @@ import (
 type PromptHandler struct {
 	moduleconfig.Parameter
 	Condition func(map[string]string) bool
+	Validate  func(string) error
```

Comment:
🎉  maybe later on we can implement some type of openapi schema validation, then user can supply that on the module side